src/browser.rs

```diff
@@ -22,23 +21,22 @@ pub fn get_mpv_message() -> Result<FF2MpvMessage, FF2MpvError> {
 }
 
 fn read_message() -> Result<String, io::Error> {
-    let mut stdin = io::stdin();
-    let mut buf: [u8; 4] = [0; 4];
-    stdin.read_exact(&mut buf)?;
+    let mut stdin = io::stdin().lock();
 
-    let length = u32::from_ne_bytes(buf);
-    let mut reader = BufReader::new(stdin.take(length as u64));
+    let mut len = 0_u32.to_ne_bytes();
+    stdin.read_exact(&mut len)?;
+    let len = u32::from_ne_bytes(len);
 
-    let mut string = String::with_capacity(length as usize);
-    reader.read_to_string(&mut string)?;
-    Ok(string)
+    let mut msg = String::with_capacity(len as usize);
+    stdin.take(len as u64).read_to_string(&mut msg)?;
```

Comment:
I think it would be more readable to do a binding for `Take` here
```suggestion
    let mut reader = stdin.take(len as u64);
    let mut msg = String::with_capacity(len as usize);
    reader.read_to_string(&mut msg)?;
```